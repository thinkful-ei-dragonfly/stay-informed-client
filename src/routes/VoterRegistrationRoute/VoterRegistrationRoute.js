import React, { Component } from 'react'
import VoterRegistration from '../../components/VoterRegistration/VoterRegistration'

export default class VoterRegistrationRoute extends Component {
  

  render() { 
    return (
      <div className="voter-reg-route">
        <VoterRegistration></VoterRegistration>
      </div>
    );
  }
}