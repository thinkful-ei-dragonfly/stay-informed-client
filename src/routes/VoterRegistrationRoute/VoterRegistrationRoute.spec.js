import React from 'react';
import { shallow } from 'enzyme'
import toJson from 'enzyme-to-json'
import VoterRegistrationRoute from './VoterRegistrationRoute'

describe(`VoterRegistrationRoute component`, () => {
  it('renders a form.VoterRegistrationRoute by default', () => {
    const wrapper = shallow(<VoterRegistrationRoute />)
    expect(toJson(wrapper)).toMatchSnapshot()
  })
})
