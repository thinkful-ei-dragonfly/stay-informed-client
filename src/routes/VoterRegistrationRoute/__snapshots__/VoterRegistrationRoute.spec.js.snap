// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`VoterRegistrationRoute component renders a form.VoterRegistrationRoute by default 1`] = `
<div
  className="voter-reg-route"
>
  <VoterRegistration />
</div>
`;
